bms-checkFunctions.js

```diff
@@ -0,0 +1,58 @@
+const {addLog} = require('./bms-log')
+const {MIN_TEMPERATURE, MAX_TEMPERATURE, MIN_SOC, MAX_SOC, MIN_CHARGING_RATE, MAX_CHARGING_RATE} = require('./bms-parameters')
+
+function checkBreach(value, lower_limit, upper_limit) {
```

Comment:
Nice abstraction to check the value of any parameter. However, to know what it returns, I have to see its implementation. Can you reflect the behavior in its name? How did you get the idea to return `1`, `-1` or `0`?

Reply:
the idea to return 1, -1 or 0 was from string comparison. strcmp(str1,str2) in C
0 | if strings are equal
1 | if the first non-matching character in str1 is greater (in ASCII) than that of str2.
-1 | if the first non-matching character in str1 is lower (in ASCII) than that of str2.